src/gc/sample/gchandletable_forsample.cpp

```diff
@@ -0,0 +1,12 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+// 
+
+// This file just includes the regular gchandletable.cpp file, but adds some
+// headers that are necessary for it to build in the GCSample project.
+
+#include "common.h"
+#include "gcenv.h"
+
```

Comment:
Why do we need this? Why can't this be done same way as the rest to keep the same simple?